lib/model-interfaces/langchain/functions/request-handler/adapters/shared/meta/llama2_chat.py

```diff
@@ -0,0 +1,64 @@
+import json
+
+from langchain.schema import AIMessage, HumanMessage
+from langchain.memory import ConversationBufferMemory
+from langchain.prompts import PromptTemplate
+
+
+Llama2ChatPrompt = """<<SYS>>
+You are an helpful assistant that provides concise answers to user questions with as little sentences as possible and at maximum 3 sentences. You do not repeat yourself. You avoid bulleted list or emojis.
+<</SYS>>
+
+{chat_history}
+
+{input} [/INST]"""
+
+Llama2ChatQAPrompt = """<<SYS>>
+Use the following conversation history and pieces of context to answer the question at the end. If you don't know the answer, just say that you don't know, don't try to make up an answer. You do not repeat yourself. You avoid bulleted list or emojis.
+<</SYS>>
+
+{chat_history}
+
+[INST] Context: {context} [/INST]
```

Comment:
This look strange and would not follow the pattern. I would just remove `[INST]` and `[/INST]` from this line.